.env.example

```
@@ -8,3 +8,14 @@ AUTH_SECRET=some-secret
 AUTH_TRUST_HOST=http://localhost:3000
 INTERNAL_API_BASE_URL=http://localhost:3000
 MOCK_AUTH=true
+
+# generate: openssl rand -hex 32
+ALTCHA_HMAC_KEY=change-this-to-random-string
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)